packages/documentation/src/stories/icons/getting-started.stories.mdx

```diff
@@ -19,101 +19,131 @@ import { BADGE } from '@geometricpanda/storybook-addon-badges';
+  console.log('Icons successfully copied!');
+} catch (err) {
+  console.error(err);
+}
 ```
 
-For more detailed information about the `post-icon` component, continue reading the <LinkTo kind="icons-components-icon--default">post-icon documentation</LinkTo>.<br/>
-A list of all icons is available <LinkTo kind="icons-search-icons--page">here</LinkTo>.
+### Configure the icons base-path
 
-### Self-hosting the icons
+Now that you have the icons available in your project, you need to tell the `icon` component where to find them.<br/>
+You can do this with two different solutions:
 
-Because depending on an external CDN may not be ideal for your project, we provide a way to self-host the SVG files on your server.<br/>
-For the following instructions, we assume that the `<post-icon>` component has already been installed as described previously.
+1. Use the **meta-tag** solution to configure the `base-path` globally for all icons on the page.
+2. Use the **base-attribute** solution, to configure the `base-path` on every `icon` component. This can also be used to overwrite the global `base-path` for a single icon.
 
-1. Install the [icons package](https://www.npmjs.com/package/@swisspost/design-system-icons) from npm.
+#### Meta-tag solution (recommended)
 
-   ```bash dark
-   npm install @swisspost/design-system-icons
-   ```
+```html dark
+<html>
+  <head>
+    <meta name="design-system-settings" data-post-icon-base="/post-icons" />
+  </head>
+</html>
+```
 
-2. In order for your project to access the installed icons, you need to copy them from our package to a `public` assets folder (this folder may have a different name in your project). All icons are available from `@swisspost/design-system-icons/public/post-icons/{icon-name}.svg`.
+#### Base-attribute solution
 
-3. To let the `<post-icon>` component know where to find your self-hosted icons, set the following meta tag in the `head` section of your page.
+```html dark
+<post-icon name="1000" base="/base-path/to/your/own/icon-folder"></post-icon>
+```
 
-   ```html dark
-   <html lang="en">
-     <head>
-       <meta name="design-system-settings" data-post-icon-base="/your/path/to/post-icons/" />
-       ...
-     </head>
-   </html>
-   ```
+## Usage of the CDN approch
 
-If you don't have access to the `head` section or if you want to set a different path for just a few icons, you can also set the base path at the icon level:
+There is one last thing we want to tell you about.
 
-```html dark
-<post-icon base="/your/path/to/post-icons/" name="1001"></post-icon>
-```
+You can also only install and set up the **components** package and use the `icon` component without any of the before descriped steps.
+In this case, the component will serve the icons from a cdn.
+
+<p class="alert alert-warning alert-md">
+  At least for now, we don't recommend this solution because of performance and cross-origin issues.
+  <br />
+  Use this approch at your own risk!
+</p>
```

Comment:
```suggestion
All icons are also available from a CDN, which is the default if you don't configure a base path for your icons and don't serve them locally. When using the CDN, please note that you need to configure your CORS policy to enable resource loading from unpkg.com. 

<p class="alert alert-warning alert-md">
  Using the CDN is not recommended for production because of higher latency when loading from the CDN, use it for quickly testing something or for prototyping.
</p>
```